server/genserver.go

```diff
@@ -460,6 +481,24 @@ func (h handler) {{.Op}}Handler(ctx context.Context, w http.ResponseWriter, r *h
 		return
 	}
 
+	{{if .HasPaging}}
+	if !swag.IsZero(nextPageID) {
+		{{if .SingleStringPathParameter}}
+		{{.InputVarName}} = nextPageID
+		w.Header().Set("X-Next-Page-Path", models.{{.Op}}InputPath({{.InputVarName}})
```

Comment:
indent this so the template `if` logic is easier to follow? gofmt will make it look good again